daemon/utils.py

```diff
@@ -84,6 +84,30 @@ def get_process_flags():
   return flags
 
 
+async def message_to_addon(app_id:str, message:str, destination:str='GUI', level:str='INFO', duration:int=5):
```

Comment:
it may make it a bit more unclear where the message is coming from as opposed to errors that are directly in the bags, but it seems good to me because it is very universal. I think they've already added some message-only tasks?

Reply:
@vilemduha The thing here is that we want to continue with the task (at least that was what I have understood from our call). If we set the task.error() and we continue with the task, it leads to unstable results:

- the error message is reported, if add-on asks for reports soon enough
- if takes more time for add-on, the message might get owerwritten by task.change_progress(0, "Waiting in queue")

I also thought we have added some tasks for messages, but could not find it :D